### PR TITLE
OAuth example

### PR DESCRIPTION
**Pull request by jeremyruppel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-APIKEY=12345
-APISECRET=98765
-```
+These examples require an API key and secret.
 
-Replace the placehodler values with your api key and api secret.
+1. Apply for an API key here: https://www.flickr.com/services/apps/create/
+  - If you already have a registered application, you can find your API key and secret here: https://www.flickr.com/services/apps/by/me
+2. In this repository, copy `.env.example` to `.env`.
+3. Replace the placehodler values in `.env` with your API key and secret.
```
Comment (pdokas) [inline README.md]: placehodler → placeholder

Comment (jeremyruppel) [inline README.md]: 🌴 🙏 📦

**Comment by alex-seville** [comment]
Nice.

File: oauth/index.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		res.redirect('/login');
+	}
+});
+
+app.get('/login', function (req, res, next) {
+	// Create a new OAuth instance
+	var oauth = new Flickr.OAuth(consumerKey, consumerSecret);
+
+	// Request a new token, providing our callback url
+	// TODO perms=?
```
Comment (pdokas) [inline oauth/index.js]: I’d vote yes on demonstrating how perms works here. Maybe even just by specifying the default read perms just so it's clear how you’d request elevated perms.

**Comment by jeremyruppel** [comment]
I'll merge this in for now since the SDK doesn't currently support perms, but I'm going to add that in right quick and will update this example.

**Pull request merged**